zummon/src/commonMain/kotlin/companysurvey/TimeSeries.kt

```diff
@@ -48,9 +48,30 @@ data class TimeSeries (
         }
     }
 
-    fun hasFullYear(year: Int): Boolean =
-        this.start <=  Instant.parse("$year-01-01T00:00:00+01:00")
-                && this.calculateEnd() >= Instant.parse("${year+1}-01-01T00:00:00+01:00")
+    fun hasFullYear(year: Int? = null): Boolean {
+        var workingYear = year ?: this.start.toLocalDateTime(TimeZone.of("Europe/Amsterdam")).year
```

Comment:
I get this error

```
DateTimeException: unsupported ZoneId:Europe/Amsterdam

captureStack@http://localhost:3000/@fs/app/build/js/packages/zero-zummon/kotlin/kotlin-kotlin-stdlib.mjs?t=1731934795453:2146:23
IllegalTimeZoneException_init_$Create$_0@http://localhost:3000/@fs/app/build/js/packages/zero-zummon/kotlin/Kotlin-DateTime-library-kotlinx-datetime.mjs?t=1731934795443:609:15
@http://localhost:3000/@fs/app/build/js/packages/zero-zummon/kotlin/Kotlin-DateTime-library-kotlinx-datetime.mjs?t=1731934795443:5755:15
@http://localhost:3000/@fs/app/build/js/packages/zero-zummon/kotlin/zero-zummon.mjs?t=1731934795453:4804:90
```

see https://github.com/Kotlin/kotlinx-datetime?tab=readme-ov-file#note-about-time-zones-in-js